.github/workflows/test.yml

```diff
@@ -1,21 +0,0 @@
-name: test
```

Comment:
In stead of removing this file, should we just merge the others into this one? (slightly on the fence if "Go" is very descriptive as a name)

Reply:
Can do, its more than just "test", and the github example uses "go". I'm happy with either though.